osbs/utils.py

```diff
@@ -206,7 +206,8 @@ def subprocess_check_output_with_retry(args, max_retries=0, backoff_factor=0):
             if counter < max_retries:
                 backoff = backoff_factor * (2 ** counter)
                 if backoff > 0:
-                    logger.info("retrying command '%s' in backoff seconds:\n '%s'", args, ex.output)
+                    logger.info("retrying command '%s' in %s second(s):\n '%s'",
```

Comment:
This looks awkward to me. Why not just the simpler `"... in %ss:\n '%s'"`? "1s", "30s" etc are not even a compromise, they are just how those durations are written.